File: app/src/main/java/com/keelim/nandadiagnosis/help/activity/PleaseActivity.java
package com.keelim.nandadiagnosis.help.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

import com.keelim.nandadiagnosis.R;

public class PleaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_please);
        Toast.makeText(this, "문의사항 메뉴 입니다.", Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/keelim/nandadiagnosis/view/setting/PleaseActivity.kt
package com.keelim.nandadiagnosis.view.setting

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.snackbar.Snackbar
import com.keelim.nandadiagnosis.R
import kotlinx.android.synthetic.main.activity_please.*

class PleaseActivity : AppCompatActivity(R.layout.activity_please) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setSupportActionBar(toolbar)
        toolbar_layout.title = title
        fab.setOnClickListener { view ->
            Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                    .setAction("Action", null).show()
        }
    }
}